tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+author: ghidora77, benamie
+tags: remote-sensing, time-series, satellite
+date_published: 2022-02-01
+---
+<!--
+Copyright 2021 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Lab 05 - Time Series Modeling
+
+## Overview
+
+The purpose of this lab is to establish a foundation for time series analysis on remotely sensed data. You will be introduced to the fundamentals of time series modeling, including decomposition, autocorrelation and modeling historical changes. At the completion of this lab, you will be able to build an explanatory model for temporal data which can be used in many different avenues of research.
+
+This lab is a segment of the [remotesensing.dev](https://www.remotesensing.dev/docs/intro), designed as six-part series on using Google Earth Engine for research in Social Science and economics. 
+
+## Background
+
+One of the paradigm-changing features of Earth Engine is the ability to access decades of imagery without the previous limitation of needing to download, organize, store and process this information. For instance, within the Landsat image collection we can access imagery back to 1972, allowing us to look at an area to visualize and quantify how much it's changed over time. With Earth Engine, Google maintains the data and offers it's computing power for processing - users can access tens or hundreds of time-sequenced images and quantify change across decades. 
+
+To explain the concepts of time series modeling, let's begin with a dataset that illustrates what we are trying to do. The line chart below references electricity production over thirty years, with one distinct data point per month. What can we observe?
+
+1. Production tends to increase each year - In this case, it appears to level out after 2010, but there is a general trend upwards. 
+
+2. Within each yearly cycle, we see that there is a sharp peak in June and July, and a trough in October and December. An annual, 12-month cycle is specifically referred to as 'seasonality', although there can be other cyclical time periods (ex., a housing market in a specific area may see a recurring pattern in house prices that occurs roughly every 7 years)
+3. Finally, the magnitude of the difference between each yearly peak and trough increases over time as well. 
+
+![Time Series Example](./im/im_05_01.png)
```

Comment:
Regarding image path, they should be relative to the .md file. Please remove the leading directories in the image paths.

`![Time Series Example](im_05_01.png)`